BlockSettleUILib/LoginWindow.cpp

```diff
@@ -21,9 +21,17 @@
 #include "ZmqContext.h"
 #include "ui_LoginWindow.h"
 #include "FutureValue.h"
+#include "QBitmap"
 
 namespace {
    const auto kAutheIdTimeout = int(BsClient::autheidLoginTimeout() / std::chrono::seconds(1));
+   const auto kProdTitle = QObject::tr("Login Into BlockSettle");
+   const auto kTestTitle = QObject::tr("Test Environment Login");
+
+   const auto kCreateAccountProd = QLatin1String("Get a BlockSettle Account");
+   const auto kCreateAccountTest = QLatin1String("Create your Test Account");
```

Comment:
I believe these strings should be translated, too.

Reply:
Thank you, changed